presto-main/src/test/java/io/prestosql/security/TestFileBasedSystemAccessControl.java

```diff
@@ -248,6 +383,18 @@ public void testRefreshing()
                 });
     }
 
+    @Test
+    public void testAllowIsRequired()
+    {
+        assertThrows(IllegalArgumentException.class, () -> newAccessControlManager(createTestTransactionManager(), "catalog_allow_unset.json"));
+    }
+
+    @Test
+    public void testAllowInvalidValue()
```

Comment:
"test allow invalid value" sounds like you're testing that invalid value is allowed, whereas it's to the contrary

renaming "Allow" to "AllowMode" would help

```suggestion
    public void testAllowModeInvalidValue()
```

for consistence rename `testAllowIsRequired` to `testAllowMode...`